package/wpe/wpeframework-cdmi/wpeframework-cdmi-playready-nexus-svp/wpeframework-cdmi-playready-nexus-svp.mk

```diff
@@ -4,9 +4,9 @@
 #
 ################################################################################
 
-WPEFRAMEWORK_CDMI_PLAYREADY_NEXUS_SVP_VERSION = R1
```

Comment:
I assume we used a release tag in the RDK Central repo? Let's later look if we can have somthing similar in the Gerrit repo, but for now I assume the hash is the latest from Gerrit which should be ok